### PR TITLE
Add rosettasciio recipe

### PR DESCRIPTION
**Pull request by ericpre**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [n/a] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [n/a] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/rosettasciio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  summary: Reading and writing scientific file formats
+  dev_url: https://github.com/hyperspy/rosettasciio
+  license: GPL-3.0-or-later
+  license_file: COPYING.txt
+
+extra:
+  recipe-maintainers:
+    - ericpre
+    - francisco-dlp
+    - jlaehne
```
Comment (ericpre) [inline recipes/rosettasciio/meta.yaml:61]: @francisco-dlp, @jlaehne, can you please confirm that you are happy to be maintainer of this recipe?

Comment (jlaehne) [reply recipes/rosettasciio/meta.yaml:61]: Sure :-)

Comment (francisco-dlp) [reply recipes/rosettasciio/meta.yaml:61]: Yes, I am happy to be maintainer of the recipe.

**Comment by francisco-dlp** [comment]
LGTM!

**Comment by ericpre** [comment]
Friendly ping @conda-forge/help-python-c, thank you!

**Comment by carterbox** [comment]
Great job on the recipe!

**Comment by ericpre** [comment]
Thank you @carterbox for the review!

**Pull request merged**